packages/build-info/package.json

```diff
@@ -4,11 +4,11 @@
   "description": "Build info utility",
   "type": "module",
   "exports": {
-    ".": "./lib/index.js"
+    ".": "./lib/src/index.js"
   },
-  "browser": "./lib/index.js",
-  "main": "./lib/index.js",
-  "types": "./lib/index.d.ts",
+  "browser": "./lib/src/index.js",
+  "main": "./lib/src/index.js",
+  "types": "./lib/src/index.d.ts",
```

Comment:
Should we set `"rootDir": "src"` in the tsconfig so that we do not create the unnecessary `/lib/src` folder?

Reply:
Implemented in https://github.com/netlify/build/pull/4876/commits/6561b05307bde3490e54d00197aa8f75a97a04c9